src/renderer.ts

```diff
@@ -125,7 +129,9 @@ const renderSlide = async (
 ) => {
   slide.hidden = node.hidden;
   if (node.backgroundColor) slide.background = { fill: node.backgroundColor };
-  if (node.backgroundImage) slide.background = { path: node.backgroundImage };
+  if (node.backgroundImage) {
+      slide.background = { [node.backgroundImage.kind]: node.backgroundImage[node.backgroundImage.kind] };
+  }
```

Comment:
If we want to keep the contract closer to the pptxgen, https://gitbrent.github.io/PptxGenJS/docs/usage-slide-options.html, we could also skip the `kind` property and propagate some changes like below: 

```suggestion
  if (node.backgroundImage) {
     slide.background =  node.backgroundImage['data' in node.backgroundImage ? 'data' : 'path']
  }
```

Reply:
How would that work? I thought the pptxgen `background` property only accepts an object